docs/core/testing/microsoft-testing-platform-extensions-test-reports.md

```diff
@@ -27,3 +27,35 @@ The available options as follows:
 | `--report-trx-filename` | The name of the generated TRX report. The default name matches the following format `<UserName>_<MachineName>_<yyyy-MM-dd HH:mm:ss>.trx`. |
 
 The report is saved inside the default _TestResults_ folder that can be specified through the `--results-directory` command line argument.
+
+## Azure DevOps reports
+
+Azure DevOps report plugin enhances test running for developers that host their code on GitHub, but build on Azure DevOps build agents. It adds additional information to failures to show failure directly in GitHub PR.
+
+![](./media/test-azdoreport-failure.png)
+
+The extension is shipped in [Microsoft.Testing.Extensions.AzureDevOpsReport](https://nuget.org/packages/Microsoft.Testing.Extensions.AzureDevOpsReport) package.
+
+> [!IMPORTANT]
+> The package is shipped with Microsoft .NET library closed-source free to use licensing model.
```

Comment:
It's not closed-source. I see this is a copy from TrxReport, but feels to me like the TrxReport note is also wrong?

Reply:
yeah, that is where I copied that from. I misread that note.